spec/javascripts/components/cookie-banner-spec.js

```diff
@@ -1,46 +1,114 @@
 /* eslint-env jasmine, jquery */
 /* global GOVUK */
 
-var container
+var container, element;
 
 var GOVUK = window.GOVUK || {};
 
-describe('Cookie banner', function () {
+describe('New cookie banner', function () {
```

Comment:
If this is making the new cookie banner into the regular cookie banner, shouldn't this be called `Cookie banner`?